portfolio/src/main/webapp/script.js

```diff
@@ -28,3 +28,9 @@ function change() {
         counter = 0;
     }
 }
+
+function getHello() {
+    fetch('/data').then(response => response.text()).then((quote) => {
```

Comment:
Quick note on style while I'm here - the indentation is a bit funky since the fetch line is on the same indentation level as the document.getElementById line.